mappings/markdown/Duo/1.1.0/duo-authentication-logs/authentication/README.md

```diff
@@ -0,0 +1,50 @@
+# Event Dossier: Duo Authentication Logs to OCSF class Authentication
```

Comment:
Hi there, thank you so much for doing this. I'm a software engineer at Duo Security and have also mapped our authlogs to OCSF schema, and wanted to share with you as I see there are some differences here: 
https://docs.google.com/document/d/1nBKz4ni88HHjPdFu7SSop0qVwZjkkz7GqvZHeBu23zY/edit

We don't have the mapping published publicly yet so just throwing it in this crude google doc link, as it's a little much for a git comment. I'd love to chat with you further about these differences and how I can contribute to this PR/mapping! 

cc: @alanisaac

Reply:
One item to add to the above: Duo a part of Cisco (and we identify it as the `metadata.product.vendor_name`), and so regardless of other changes in the logs above, I'd group Duo mappings under the Cisco folder.

Reply:
In the weekly collaboration OCSF collaboration call, it was suggested that maybe we could pull the Duo mapping out, collaborate on it together, and then contribute it back separately.  I gather this is what was done with AWS as well.

Reply:
It would be great if you can review other Cisco mappings that are being submitted via this PR, that will help ensure we are not creating/publishing divergent mappings for the same source.

Reply:
@brandikuritz sorry it took so long to respond!
Thanks for sharing the mapping, exciting to see more and more vendors getting on board.
I moved the mappings under Cisco and added parts of the Actor fields which we've missed and could definitely be valuable here.